Cargo.toml

```diff
@@ -21,9 +21,11 @@ A crate for interacting with the Alpaca API.
 include = ["src/**/*", "LICENSE", "README.*", "CHANGELOG.*"]
 
 [dependencies]
+async-compression = {version = "0.3.12", optional = true}
 async-trait = "0.1.51"
 chrono = {version = "0.4.19", features = ["serde"]}
 futures = {version = "0.3", default-features = false}
+futures-util = {version = "0.3.21", optional = true}
```

Comment:
I don't think we need any additional dependencies aside from `async-compression`, to be honest. `futures-util` should just be a subset of `futures`, which we already depend on.

Reply:
Done.